src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -419,9 +419,7 @@ public void evaluate() throws Throwable {
                     Thread.sleep(100);
                 }
                 LOGGER.info("agent back online");
-                /* TODO JENKINS-50504 still fails:
                 assertWorkspaceLocked(computer, workspacePath);
```

Comment:
It's not totally clear to me what the purpose of the PR is. Did this just fail intermittently before https://github.com/jenkinsci/jenkins/pull/3955? I was expecting to see some existing test become unignored or for a new test to be added.

Reply:
The part of the test uncommented here fails consistently without the core patch.

Reply:
Oh, duh, I was reading the comments as line comments rather than as the start and end of a block comment... Thanks for clearing that up!